### PR TITLE
Add tool to convert TIPSY ICs to GENGA format

### PR DESCRIPTION
**Pull request by spencerw**

### CODE REVIEW

File: bin2genga.c [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+main()
+{
+    struct dark_particle *dark_particles, *dp, *lastdp;
+    struct dump header;
+    
+    while(1) 
+      {
+	if(fread((char *)&header,sizeof(header),1,stdin) == 0)
+	  break;
+	if(header.ndark != 0) {
```
Comment (trquinn) [inline bin2genga.c:23]: It looks like you assume that nSph == 0 and nStar == 0.  You should check and assert on that.

File: bin2genga.c [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+		return -1;
+	    }
+	}
+    
+	fread((char *)dark_particles,sizeof(struct dark_particle),
+			 header.ndark,stdin) ;
+
+	lastdp = dark_particles + header.ndark ;
+
+    for(dp=dark_particles; dp< lastdp; dp++) {
```
Comment (trquinn) [inline bin2genga.c:37]: Could you clean up the indentation?

**Review by trquinn — CHANGES_REQUESTED** [review]
Small changes requested.

**Comment by spencerw** [comment]
Will do!

I'm also realizing that this tool doesn't seem to work with the ChaNGa output files (only the IC files that I create with ascii2bin). Is there another conversion that needs to be done to the ChaNGa outputs first? I've been using iBinaryOut = 0.

**Comment by spencerw** [comment]
I see, let's leave "totipnat" to take care of that step and have this tool function similarly to "bin2ascii".

I realize I probably should have named this branch "bin2genga". I don't think there's a way to rename a branch after opening a pull request, so let me know if you want me to create a new one, or just merge as-is.

**Comment by trquinn** [comment]
This is superceded by pr #5

**Pull request closed without merging**